en/advanced/gimbal_control.md

```diff
@@ -1,20 +1,32 @@
 # Gimbal Control Setup
 
-PX4 contains a generic mount/gimbal control driver with different input and output methods. Any input can be selected to drive any output.
+If you want to control a gimbal with a camera (or any other payload) attached to
+the vehicle, you need to configure how you want to control it and how PX4 can
+command it. This page explains the setup.
 
-First, make sure the driver runs, using `vmount start`, then configure its parameters.
+PX4 contains a generic mount/gimbal control driver with different input and
+output methods. The input defines how you control the gimbal: via RC or via
+MAVLink commands (for example in missions or surveys). The output defines how
+the gimbal is connected: some support MAVLink commands, others use PWM
+(described as AUX output in the following). Any input method can be selected to
+drive any output. Both have to be configured via parameters.
 
 ## Parameters
-The parameters are described in [src/drivers/vmount/vmount_params.c](https://github.com/PX4/Firmware/blob/master/src/drivers/vmount/vmount_params.c). The most important ones are the input (`MNT_MODE_IN`) and the output (`MNT_MODE_OUT`) mode. By default, the input is disabled. Any input method can be selected to drive any of the available outputs.
+[These parameters](../advanced/parameter_reference.md#mount) are used to setup
+the mount driver. The most
+important ones are the input (`MNT_MODE_IN`) and the output (`MNT_MODE_OUT`)
+mode. By default, the input is disabled and the driver does not run. After
+selecting the input mode, reboot the vehicle so that the mount driver starts.
 
-If a mavlink input mode is selected, manual RC input can be enabled in
-addition (`MNT_MAN_CONTROL`). It is active as long as no mavlink message is received yet, or mavlink explicitly requests RC mode.
+If the input mode is set to `AUTO`, the mode will automatically be
+switched based on the latest input. To switch from mavlink to RC, a large stick
+motion is required.
 
+## Configure the gimbal mixer for AUX output
 
-
-### Configure the gimbal mixer for AUX output
-
-The gimbal uses the control group #2 (see [Mixing and Actuators](../concept/mixing.md)). This is the mixer configuration:
+If the `MNT_MODE_OUT` parameter is set to `AUX`, a mixer file has to be configured.
```

Comment:
My knowledge here is actually only theoretical, as I have never set this up myself on real HW :)

I addressed your comments, see new commits.